default.nix

```diff
@@ -33,6 +33,8 @@ in
       # This is kind of round-about but it seems to be the best way to
       # convince mach-nix to use a specific package for a specific dependency.
       tahoe-lafs = "nixpkgs";
+
+      cryptography = "nixpkgs";
```

Comment:
The packaging fix was mostly this.  mach-nix can't find a new enough cryptography to satisfy the newer autobahn using only the "sdist" and "wheel" providers specified above.  "nixpkgs" lets it get one that is just barely new enough (but still quite old, it would be nice to fix the issue that prevents it from getting a more recent version...)